README.rst

```diff
@@ -30,7 +37,9 @@ Requirements
 ------------
 
 * Python 3.10 or above
-* Django 3.2 or newer
+* Django 3.2 or above
```

Comment:
What's the reason django 3.2 still needs to be supported? It's been EOL for a while